Userland/Libraries/LibGfx/ImageFormats/CCITTDecoder.cpp

```diff
@@ -420,20 +420,42 @@ struct CCITTStatus {
     bool has_reached_eol { false };
 };
 
-ErrorOr<void> ensure_invalid_result_is_actually_eol(BigEndianInputBitStream& input_bit_stream, InvalidResult partially_read_eol)
+ErrorOr<void> ensure_invalid_result_is_actually_eol(BigEndianInputBitStream& input_bit_stream, InvalidResult partially_read_eol, Group4Options const& options)
 {
     if (partially_read_eol != 0)
         return Error::from_string_literal("CCITTDecoder: Unable to find the correct mode");
 
     auto const remaining_eol = TRY(input_bit_stream.read_bits(5));
-    if (remaining_eol != 1) {
+    if (options.has_end_of_block == Group4Options::HasEndOfBlock::Yes
```

Comment:
nit: could mention in commit description that this only happens if the "has end of block" option is set